src/spriteset_map.cpp

```diff
@@ -40,6 +41,13 @@ Spriteset_Map::Spriteset_Map() {
 		character_sprites.push_back(EASYRPG_MAKE_SHARED<Sprite_Character>(i->second.get()));
 	}
 
+	Game_Vehicle* vehicle;
+	for (int i = 1; i <= 3; ++i) {
+		vehicle = Game_Map::GetVehicle((Game_Vehicle::Type) i);
+		if (vehicle->IsInCurrentMap())
```

Comment:
If the vehicle is off map, and then an event puts it on map with Set Vehicle Location, the vehicle will be invisible. You can test it in eg. OFF at a pedalo recovery point: use `--start-map-id 53 --start-position 15 9` (you'll be directly below the Elsen), kill him, and call a pedalo.